iBudget/fund/models.py

```diff
@@ -88,3 +88,15 @@ def filter_by_data(value, start_date, finish_date, fund):
             finish_date=finish_date,
             fund=fund)
         return goals
+
+    @staticmethod
+    def filter_goals_by_fund_id(fund_id):
+        """
+        Args:
+            fund_id(int): Current session fund`s id.
+        Returns:
+            List of Goal objects .
+
+        """
+        filter_goal_by_fund = FinancialGoal.objects.filter(fund=fund_id)
```

Comment:
Local variable `filter_goal_by_fund` is not really needed here. Please rename it to just `goals` if you would like to leave it.

Reply:
So... looking into the code below, you do not really need to return list of objects here because you just check if something was found. I would better use here something like 
```
return FinancialGoal.objects.filter(fund=fund_id).exists()
```

Reply:
And rename it to `has_goals` or something similar.